src/components/NewClusterModal.js

```diff
@@ -205,10 +215,16 @@ export const NewClusterModal = withModalDrawer({ width: 675 })(class NewClusterM
     ])
 
     const bottomButtons = () => h(Fragment, [
-      div({ style: { flexGrow: 1 } }),
       div(
-        { style: { display: 'flex', marginTop: '1rem' } },
+        { style: { display: 'flex', marginTop: '3rem', marginBottom: '1rem' } },
         [
+          currentCluster && div([
+            h(ButtonSecondary, {
+              onClick: () => this.setState({ isDeleteView: true }),
+              tooltip: 'Delete runtime configuration',
+              'aria-label': 'Delete runtime configuration'
```

Comment:
This tooltip and label seem redundant, since they're not really adding much information beyond the existing button text. Can we remove them?